gulp-tasks/utils/constants.js

```diff
@@ -18,4 +15,12 @@ module.exports = {
   // TODO: Change to v3.0.0 when v3 is launched to avoid doing anything
   // with prelease versions.
   MIN_RELEASE_TAG_TO_PUBLISH: 'v3.0.0-alpha',
+  GITHUB_OWNER: 'GoogleChrome',
+  GITHUB_REPO: 'workbox',
+
+  // This is the namespace of the workbox module.
+  NAMESPACE_PREFIX: 'google.workbox',
```

Comment:
I'm not sure what the best place to have this discussion is at this point (happy to move to the tracking issue or the Google Doc), but what are the benefits of `google.workbox.` vs. `workbox.` as the namespace prefix?